Updater.js

```diff
@@ -259,7 +278,7 @@ define(['./util/lang', './Variable'], function (lang, Variable) {
 	ListUpdater.prototype.renderUpdate = function (newValue, element) {
 		var container
 		var each = this.each
-		var thisElement = this.element
+		var thisElement = this.elements[0]
```

Comment:
oops, `this.element` should suffice. in any case, probably not worthwhile to support multiple targets for lists...